scripts/uninstall.sh

```diff
@@ -44,22 +43,43 @@ delete_app_config_dir() {
 
 delete_app() {
 
-    logger "Killing Any $APP_NAME Processes ..."
-    pkill -f "$APP_NAME"
+    # Find the PID(s) of the application
+    PIDS=$(pidof "$APP_NAME") || true
+
+    if [ -n "$PIDS" ]; then
+        echo "Found $APP_NAME with PID(s): $PIDS. Attempting to kill..."
+
+        # Attempt to terminate the process gracefully
+        kill $PIDS 2>/dev/null || echo "Graceful kill failed..."
+
+        # Wait for a short period to allow graceful shutdown
+        sleep 2
+
+        # Check if the process is still running
+        PIDS=$(pidof "$APP_NAME") || true
+        if [ -n "$PIDS" ]; then
+            echo "Process still running. Force killing..."
+            kill $PIDS 2>/dev/null || echo "Graceful kill failed..."
```

Comment:
this is also duplicated by mistake

Reply:
yup , it supposed to be `kill -9`
going to fix ...